ocs_ci/templates/workloads/smallfile/SmallFile.yaml

```diff
@@ -11,7 +11,8 @@ spec:
   test_user: homer_simpson
   clustername: place-holder
   elasticsearch:
-    #server: 10.0.144.152 <- debuging server
+    # v This is debug server v
```

Comment:
typo?

Reply:
No, this is not type, the remark is for the line 15 - so in case of debugging, you can only replace the # from the debug server to prod server without the need to change the end of the line.